webcompat/static/js/lib/bugform.js

```diff
@@ -221,49 +221,57 @@ function BugForm() {
 
   this.showSimilarBugs = function() {
     if (!$("body").data("username")) {
-      // Not login
+      // Not logged in
       return;
-    }
+    } else {
+      if (this.suggestDelay) {
+        clearTimeout(this.suggestDelay);
+      }
 
-    if (this.suggestDelay) {
-      clearTimeout(this.suggestDelay);
-    }
-    var bugform = this;
-    this.suggestDelay = setTimeout(function() {
-      try {
-        var url = new URL(bugform.urlField.val());
-        if (url.hostname.length > 7) {
-          var query = `https://api.github.com/search/issues?page=1&per_page=10&stage=all&sort=created&q=${url.hostname}+in:title+repo%3Awebcompat%2Fweb-bugs&order=desc`;
-
-          fetch(query)
-            .then(function(response) {
-              var contentType = response.headers.get("content-type");
-              if (contentType && contentType.includes("application/json")) {
-                return response.json();
-              }
-            })
-            .then(function(issues) {
-              if (issues.items) {
-                $("#bugListWrapper").show();
-                $("#bugList").empty();
-                for (var item in issues.items) {
-                  var issue = issues.items[item];
-                  $("#bugList").append(
-                    `<li><a href='https://webcompat.com/issues/${issue.number}' target='_blank'>#${issue.number} - ${issue.title}</a></li>`
-                  );
+      var bugform = this;
+
+      this.suggestDelay = setTimeout(function() {
+        try {
+          // url got removed the http protocol, so the url works as a keyword. Shortest url.hostname is something like www.
```

Comment:
I don't understand this comment. Can it be clarified?

Reply:
> url got removed the http protocol, so the url works as a keyword. 

The type URL normalizes the value of the `#url` input. Here for it splits the value in sections.`url.hostname` is the hostname and can be used as a keyword.
`https://test` -> `test`

`test` would be the keyword here.

> Shortest url.hostname is something like www.

We added a restriction to the min. char length, so the functionality to look for a duplicate issue does not start unnecessarily and we don't waste requests or get blocked by Github API.
So, we were thinking about the shortest possible URL people would probably use... 

The usecase `www.` is something that sounds reasonable as well as `x.io`, so we chose the min char of 4.